docs/modules/gcloud.md

```diff
@@ -0,0 +1,51 @@
+# GCloud Module
+
+Testcontainers module for the Google's [Cloud SDK](https://cloud.google.com/sdk/).
+
+Currently, the module supports `datastore`, `firestore`, `pubsub` and `spanner` emulators. In order to use it, you should use the following classes:
+
+* DatastoreEmulatorContainer
+* FirestoreEmulatorContainer
+* PubSubEmulatorContainer
+* SpannerEmulatorContainer
+
+## Usage example
+
+Running GCloud as a stand-in for Google Firestore during a test:
+
+```java
```

Comment:
Please could you make this an executable example by using the `codeinclude` macro that we have set up? (please see [here](https://github.com/testcontainers/testcontainers-java/blob/master/docs/modules/databases/mongodb.md#L10-L12) for an example)